src/test/java/uk/gov/ons/ctp/response/notify/message/ActionInstructionReceiverImplTest.java

```diff
@@ -116,6 +116,7 @@ public void testProcessBRESInstructionNotifyThrowsException() throws Notificatio
       actionInstructionReceiver.processInstruction(
           ObjectBuilder.buildActionInstruction(
               "9a5f2be5-f944-41f9-982c-3517cfcfef3c", "Joe,Blogg,07742994131,tester@gmail.com", true));
+      // Fail the test if an exception is not thrown
```

Comment:
junit 5 looks to have a nicer way to handle expecting exceptions https://stackoverflow.com/a/40268447/4695665

Reply:
Do we have the version of Junit pinned somewhere?

Reply:
I can't see where it's defined. Maybe surefire is controlling this as it looks like surefire uses 4.12 and that's what's included